packages/ts-invariant/src/invariant.ts

```diff
@@ -25,17 +25,29 @@ export function invariant(condition: any, message?: string | number) {
   }
 }
 
-function wrapConsoleMethod(method: "warn" | "error") {
+const verbosityLevels = ["log", "warn", "error", "silent"] as const;
```

Comment:
This `as const` annotation makes TypeScript infer a tuple type of `["log", "warn", "error", "silent"]` for this array, rather than `string[]`.